doc/man3/SSL_CTX_set_options.pod

```diff
@@ -306,9 +314,11 @@ L<dhparam(1)>
 The attempt to always try to use secure renegotiation was added in
 Openssl 0.9.8m.
 
+B<SSL_OP_PRIORITIZE_CHACHA> was assed in OpenSSL 1.1.1.
```

Comment:
This doesn't comply with Oxford dictionary, which offers to meanings, to call and to act as.

Reply:
I mean two meanings for the verb.

Reply:
In my own defense, 's' and 'd' are next to each other on the keyboard, and I did type a real word. :)